exasol_advanced_analytics_framework/event_handler/event_handler_result.py

```diff
@@ -11,7 +11,16 @@ class EventHandlerReturnQuery:
 
 
 @dataclass()
-class EventHandlerResult:
-    return_query: Optional[EventHandlerReturnQuery]
+class EventHandlerResultBase:
     status: str
     query_list: List[str]
```

Comment:
The EventHandlerResultFinished doesn't need a query_list, we only create a query_list in the Python UDF to cleanup the temporary tables